packages/web/src/js/Collapse.ts

```diff
@@ -0,0 +1,197 @@
+import BaseComponent from './BaseComponent';
+import { instanceAutoloader } from './utils/ComponentFunctions';
+import $ from './dom/SelectorEngine';
+import EventHandler from './dom/EventHandler';
+
+const NAME = 'collapse';
+const NAME_ARIA_EXPANDED = 'aria-expanded';
+const NAME_ARIA_CONTROLS = 'aria-controls';
+const NAME_DATA_TARGET = 'data-target';
+const CLASSNAME_EXPANDED = 'is-expanded';
+const CLASSNAME_COLLAPSED = 'is-collapsed';
+const CLASSNAME_HIDDEN = 'is-hidden';
+const BREAKPOINT_MOBILE = 0;
```

Comment:
Once again, these breakpoints can change in every product design. Responsiveness should be handled by CSS.